### PR TITLE
Adding troubleshooting info

### PR DESCRIPTION
**Pull request by sishida**
Signed-off-by: Sarah Ishida <sishida@us.ibm.com>

Updates as part of issue https://github.com/eclipse/codewind/issues/149.

### CODE REVIEW

**Comment by sishida** [comment]
@thisguy-1 I have added the link.

@tetchel Do you have additional info, especially about the "Installing the Codewind containers without a prompt" section?

**Comment by tetchel** [comment]
I don't know what's supposed to go there? The user should always see the prompt

**Comment by sishida** [comment]
@tetchel I probably misunderstood that we needed that section in the first place. I have removed it. If the other changes look good to go, please feel free to merge. :)

**Comment by tetchel** [comment]
I don't think I would know what "the notifcation window" refers to here. I think we should rephrase this as just retrying the installation.

> Accessing the notification window to install the Codewind containers
->
> Re-initiating the Codewind installation
or something

**Comment by sishida** [comment]
@tetchel Thanks for the feedback! Here's my latest edits in my newest commit: https://github.com/eclipse/codewind-docs/pull/116/commits/e2d29a3355409bdad0633350f507ea97b54ea591
